src/components/social-cards/race/state.js

```diff
@@ -0,0 +1,293 @@
+import React from 'react'
+import { useStaticQuery, graphql } from 'gatsby'
+import classnames from 'classnames'
+import { renderedComponent } from '~plugins/gatsby-render-components'
+import slug from '~utilities/slug'
+import Percent from '~components/pages/race/dashboard/percent'
+import { stringifyList } from '~utilities/list-formatter'
+import logo from '~images/project-logo-black.svg'
+import socialCardStyle from './state.module.scss'
+
+const State = renderedComponent(({ state, population }) => {
+  const getField = (field, type) => {
+    if (field === 'all') {
+      return null
+    }
+    if (!state[field + type]) {
+      return 0
+    }
+    return Math.round(
+      (parseInt(state[field + type], 10) /
+        (parseFloat(state[`${field}PctPop`]) * population)) *
+        100000,
+    )
+  }
+
+  let totalCases = 0
+  let totalDeaths = 0
+  Object.keys(state).forEach(field => {
```

Comment:
I think there's a property that has the `totalCases` and `totalDeaths` values w/o this type of iteration...I'm not 100% sure, though.

Reply:
Nevermind, there isn't!